src/Corax/Queries/BinaryMatch.Boolean.cs

```diff
@@ -42,6 +42,17 @@ static int AndWith(ref BinaryMatch<TInner, TOuter> match, Span<long> matches)
                 return outer.AndWith(matches.Slice(0, results));
             }
 
+            static QueryInspectionNode InspectFunc(ref BinaryMatch<TInner, TOuter> match)
+            {
+                return new QueryInspectionNode($"{nameof(BinaryMatch)} [And]",
```

Comment:
You didn't implement this for OR?

Reply:
Yes.
https://github.com/ravendb/ravendb/pull/13270/files/51fa09050f5e2500b16b71ade428465cd19ff2a3#diff-387c28ce8317892df9dd1c7dda739e8826bb775064d6871f3e5e68ebd446f7fcR171